site/public/css/diff-viewer.css

```diff
@@ -1,4 +1,3 @@
-@import url("reset.css");
```

Comment:
I don't bring the server.css in. I bring every css except that one since I don't want the background to change blue.